### ISSUE TITLE
Error: Missing scheduler size input when trying to enable dag-deploy in Astro CLI 1.15.1 for hosted trial deployment

### ISSUE THREAD

**Issue by TJaniF** [astronomer/astro-cli#1226]
**Describe the bug**

I wanted to test DAG only deploys with a hosted trial deployment (no changes to default settings). Using the Astro CLI 1.13.0 enabling DAG only deploys works as expected. After upgrading to 1.15.1 I get the following error:

```
(base) tjanif@Tamaras-Mini datasets-sdk-live % astro deployment list
 NAME        NAMESPACE                    CLUSTER                    DEPLOYMENT ID                   RUNTIME VERSION                    DAG DEPLOY ENABLED     CI-CD ENFORCEMENT     
 Sandbox     cometary-ionization-6143     Shared GCP us-central1     cli1ucwyw785995k4xhseu4i5pe     8.2.0 (based on Airflow 2.6.1)     false                  false                 
(base) tjanif@Tamaras-Mini datasets-sdk-live % astro deployment update cli1ucwyw785995k4xhseu4i5pe --dag-deploy enable

You enabled DAG-only deploys for this Deployment. Running tasks are not interrupted but new tasks will not be scheduled.
Run `astro deploy --dags` to complete enabling this feature and resume your DAGs. It may take a few minutes for the Airflow UI to update..

Error: Missing scheduler size input
(base) tjanif@Tamaras-Mini datasets-sdk-live % astro deployment list                                                  
 NAME        NAMESPACE                    CLUSTER                    DEPLOYMENT ID                   RUNTIME VERSION                    DAG DEPLOY ENABLED     CI-CD ENFORCEMENT     
 Sandbox     cometary-ionization-6143     Shared GCP us-central1     cli1ucwyw785995k4xhseu4i5pe     8.2.0 (based on Airflow 2.6.1)     false                  false                 
(base) tjanif@Tamaras-Mini datasets-sdk-live % astro version
Astro CLI Version: 1.15.1
```

Note: I don't think this is relevant but just in case I turned Docker Desktop off during this test.

**What CLI Version did you experience this bug?**

1.15.1

**This CLI bug is related to which Astronomer Platform?**

- [x] Astro
- [ ] Software
- [ ] None/Unknown

**What Operating System is the above CLI installed on?**

Apple M1 Pro 13.3.1 (22E261)

**🪜 Steps To Reproduce**

1. Create a new deployment in a hosted trial account
2. Run `astro deployment update <deployment_id> --dag-deploy enable`
3. See error

**Comment by kushalmalani** [comment]
Can you switch organizations via cli and switch back to shared org and give it a try again?

TLDR: So what happened here was when you were on 1.13.0 version, and logged into shared org, we set an attribute in config file which that version doesn’t contain. So when you upgraded and tried again, you were seeing the issue since that config was missing. If you switch orgs now and switch back to a shared org that value should be set and you should get a proper experience

This shouldnt be a problem for customers since they usually are part of a single org. This is only for us internal folks since we are part of multiple orgs within our system

**Comment by TJaniF** [comment]
💡 !

Switching organizations over to Astronomer then back to the hosted trial one fixed the issue! Thanks for your help! 😄
